src/KeyboardInputSchemes.test.js

```diff
@@ -53,7 +53,7 @@ it('Should be able to handle a complete valid sequence', () => {
 
 it('Should be able to handle a complete invalid sequence', () => {
     const completeInvalidSequence = [
-        new KeyboardEvent('keydown', { code: "KeyA", altKey: true}),
+        new KeyboardEvent('keydown', { code: "KeyZ", altKey: true}),
```

Comment:
That's funny.  Hopefully we won't have to get weird with the test cases as we keep adding keys.